Analytics/Integrations/Kahuna/SEGKahunaIntegration.h

```diff
@@ -22,6 +22,7 @@
 @property (nonatomic) NSDictionary *pushInfo;
 @property (nonatomic) UIApplicationState applicationState;
 @property (nonatomic) BOOL kahunaInitialized;
```

Comment:
@sharekris I noticed that you made these properties atomic in your change, is that strictly required?